client/js/lounge.js

```diff
@@ -628,12 +628,23 @@ $(function() {
 		return false;
 	});
 
+	// Used when resetting original height after deleting a character
+	var initialInputHeight;
+
 	var input = $("#input")
 		.history()
 		.on("input keyup", function() {
 			var style = window.getComputedStyle(this);
-			this.style.height = "0px";
-			this.offsetHeight; // force reflow
+
+			if (!initialInputHeight) {
+				// When first resizing the textarea, store its initial height for later
+				initialInputHeight = this.style.height;
```

Comment:
There's no need to store it this way. You already have it in `style.minHeight` which contains the minimum height which is normally what prevents the 0px from applying in non-retarded browsers. So we can just use this one instead.